PlanningBiblio/PlanningBiblio
# MT38395 : fix check mail config

see MT38395

TEST PLAN : 
- edit config
- change mail settings and click the button "Tester" each time you change a field
- the test should take values from the fields before the form is submitted, and show a success or error result depending these values
